app/helpers/jwt_utils.js

```diff
@@ -6,6 +6,6 @@ const {
   }
 } = require('../../config');
 
-exports.encode = payload => encode(payload, secret, 'HS256');
+exports.encode = payload => encode({ ...payload, iss: Date.now() }, secret, 'HS256');
```

Comment:
I think you wanted to set `iat` (issued at). `iss` identifies the issuer of the jwt.

Reply:
nice catch